tests/test_sampling.py

```diff
@@ -208,6 +208,20 @@ def test_linear_regression(self):
         _ = sgld.step(rng_key, init_state, data_batch)
 
 
+irmh_test_case = {
```

Comment:
I extracted this in order to have it close to the comment explaining the rationale for the proposal distribution. LMK if other format is preferred.